src/components/Table/hoc/withTableSettings/TableColumnSetup/TableColumnSetup.tsx

```diff
@@ -242,6 +246,17 @@ const mapItemDataToProps = (item: TableColumnSetupItem): ListItemCommonProps =>
     };
 };
 
+const defaultFilterItemsFn = (item: TableColumnSetupItem, value: string) => {
```

Comment:
may be it's passible to use `useListFilter` from [this](https://github.com/gravity-ui/uikit/blob/main/src/components/TreeSelect/__stories__/components/WithFiltrationAndControlsExample.tsx#L48) example and not realise filtration logic from stretch?

it is also welcome for the development of a hook component for filtering

@polikashina @amje @Raubzeug  What do you think?

Reply:
[fixed](https://github.com/gravity-ui/uikit/pull/1627/commits/05233de4c44a860d5dfe38e6318cc68bb87f607b)